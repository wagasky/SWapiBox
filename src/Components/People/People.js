import React from 'react';
import Card from '../Card/Card'

const People = (props) => {

  console.log(this.props)

  return (
    <div>
      <p>I'm People!</p>
      <Card />
    </div>
  )
}

export default People;