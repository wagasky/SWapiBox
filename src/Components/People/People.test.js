/* eslint-disable */ 

import React from 'react';
import ReactDOM from 'react-dom';
import People from './People';
import { shallow } from 'enzyme';

describe('People', () => {

  let renderedComponent;
  let mockHandleFavorite;

  const mockData = [
      {"name":"Luke Skywalker","homeworld":"Tatooine","species":"Human","population":"200000"},
      {"name":"C-3PO","homeworld":"Tatooine","species":"Droid","population":"200000"},
      {"name":"R2-D2","homeworld":"Naboo","species":"Droid","population":"4500000000"},
    ]

  it('should match snapshot', () => {
    renderedComponent = shallow(<People data={mockData} handleFavorite={mockHandleFavorite} />);

    expect(renderedComponent).toMatchSnapshot();
  });

});