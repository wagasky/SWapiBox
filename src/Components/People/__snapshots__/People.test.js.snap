// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`People should match snapshot 1`] = `
<div
  className="people-cards"
>
  <Card
    data={
      Object {
        "homeworld": "Tatooine",
        "name": "Luke Skywalker",
        "population": "200000",
        "species": "Human",
      }
    }
    key="0"
  />
  <Card
    data={
      Object {
        "homeworld": "Tatooine",
        "name": "C-3PO",
        "population": "200000",
        "species": "Droid",
      }
    }
    key="1"
  />
  <Card
    data={
      Object {
        "homeworld": "Naboo",
        "name": "R2-D2",
        "population": "4500000000",
        "species": "Droid",
      }
    }
    key="2"
  />
</div>
`;
